src/Components/Activities/Detail/ActivityDetail.js

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from 'react';
+//COMPONENTS
+import Title from '../../Title/Title';
+//MUI
+import { CircularProgress, Container, Typography } from '@mui/material';
+//In the future this API call logic should be moved.
+import axios from 'axios';
+const getActivityById = (id) => {
+  return axios.get(`http://ongapi.alkemy.org/api/activities/${id}`);
+};
+
+export default function ActivityDetail({ match }) {
```

Comment:
what this match is for?

Reply:
I use it to get the activityId from the url. If path is '/activities/3', then activityId = match.params.id = 3, so I use activityId to make the API call.